cmd/cloudx/client/command_helper_test.go

```diff
@@ -28,64 +35,92 @@ import (
 var updatedProjectConfig json.RawMessage
 
 func TestMain(m *testing.M) {
-	testhelpers.RunAgainstStaging(m)
+	if err := playwright.Install(&playwright.RunOptions{
+		Browsers: []string{"chromium"},
+	}); err != nil {
+		panic(err)
+	}
+	os.Exit(m.Run())
```

Comment:
```suggestion
	m.Run()
```

`TestMain` does the correct thing on its own.

Reply:
Interesting, didn't know they eventually fixed this. It is already for quite some time, appareltly go1.15 https://github.com/golang/go/commit/2f54081adfc967836842c96619d241378400ece6